apps/src/studio/skins.js

```diff
@@ -539,6 +540,7 @@ function loadHoc2015x(skin, assetUrl) {
   };
 
   skin.specialItemProperties = {
+    'hazard': { frames: 13, width: 100, height: 100, renderOffset: { x: 0, y: -25}, activity: 'watchActor', speed: constants.SpriteSpeed.VERY_SLOW }
```

Comment:
Seems fine to me. However, this is not "maximum speed" - it is default speed for this class of item. And when the student calls setItemSpeed - it will change the speed of all of the items of that class.